charts/victoria-metrics-operator/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## Next release
 
-- TODO
+- Renamed `cleanupCRD` to `cleanupCRD.enabled`
+- Renamed `cleanupImage` to `cleanupCRD.image`
+- Replaced `watchNamespace` string with `watchNamespaces` slice
```

Comment:
People hate such changes. We must have strong motivation to introduce it.

Reply:
updated changelog and added support of both old and new properties, marked old ones as deprecated